p2p/client.go

```diff
@@ -93,7 +94,7 @@ func (c *Client) bootstrap() error {
 		}
 		c.logger.Debug("seed", "addr", maddr.String())
 		// TODO(tzdybal): configuration param for connection timeout
-		ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+		ctx, cancel := context.WithTimeout(c.ctx, 3*time.Second)
```

Comment:
Should we extract `3*time.Second` to some documented `const` or rather `var`?

Reply:
I was thinking about configuration TBH, but for now I'll just use `const`.